front/src/pods/student/student.component.tsx

```diff
@@ -40,20 +38,9 @@ export const StudentComponent: React.FC<Props> = props => {
         <label className={innerClasses.label} htmlFor="session">
           Content
         </label>
-        <div role="log">
-          <TextareaAutosize
-            ref={textAreaRef}
-            data-testid="session"
-            id="session"
-            maxRows={30}
-            minRows={30}
-            className={innerClasses.textarea}
-            value={log ?? ''}
-            readOnly={true}
-          />
-        </div>
+        <MarkdownView value={log ?? ''} className={innerClasses.textView} />
```

Comment:
Don't remove the `<div role="log"`. It's important for a11y

Reply:
Done